README.md

```diff
@@ -366,6 +366,10 @@ A curated list of awesome inclusive events/projects/initiatives for women in the
 
     Lambda Ladies is an online community for anyone who identifies as a girl or woman and is interested in functional programming. Complete beginners are welcome.
 
+  - [Oracle Women in Technology](https://community.oracle.com/groups/otn-women-in-technology/overview)
+
+    Oracle WIT Community Group - a place to share your experience and mentor women in high tech roles. Join us as we share our knowledge, technical resources and engage with other women throughout the Oracle technical community.
```

Comment:
We don't represent them, so we can't use the word 'we' in our description. Other than that I'd say we make the text more informative over marketing, but this stands okay right now. Let me know when you've made the changes.